Tasks/ACRTaskV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -0,0 +1,67 @@
+{
+  "loc.friendlyName": "ACR Task",
+  "loc.helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?linkid=851275) or [see the ACR Task documentation](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-tasks-overview)",
+  "loc.description": "Buid and run container images using ACR tasks",
+  "loc.instanceNameFormat": "acrTask",
+  "loc.group.displayName.platform": "Platform",
+  "loc.group.displayName.advanced": "Advanced",
+  "loc.input.label.connectedServiceName": "Azure subscription",
+  "loc.input.help.connectedServiceName": "Select an Azure resource manager subscription for the deployment",
+  "loc.input.label.azureContainerRegistry": "Azure container registry",
+  "loc.input.help.azureContainerRegistry": "Select an Azure Container Registry which will be used for pulling container images and deploying applications to the Kubernetes cluster. Required for commands that need to authenticate with a registry.",
+  "loc.input.label.contextPath": "Source context path",
+  "loc.input.help.contextPath": "Path of the context to be uploaded. For release having multiple artifacts provide with artifact alias name in the path too. For example, $(System.DefaultWorkingDirectory)/_alias1 ",
+  "loc.input.label.dockerfileOrTaskFile": "Dockerfile or task file path",
+  "loc.input.help.dockerfileOrTaskFile": "Relative path to the Dockerfile or the task template/definition file path relative to the source context",
+  "loc.input.label.imageNames": "Image names",
+  "loc.input.help.imageNames": "A list of the names of the Docker images in separate lines to build and push.",
+  "loc.input.label.arguments": "Arguments",
+  "loc.input.help.arguments": "The collection of build arguments to be used when executing a docker build. Format: --build-arg arg1=value1 --build-arg arg2=val2",
+  "loc.input.label.valuesFilePath": "ValuesFilePath",
+  "loc.input.help.valuesFilePath": "The task values/parameters file path relative to the source context. Valid for yaml files",
+  "loc.input.label.os": "Operating System",
```

Comment:
keep  consistency with label. either change to Operation system or change other labels to  format 'Operating System'.